README.md

```diff
@@ -1,84 +1,93 @@
 
-| Arguments       | Usage       | Description                                             |
-|-----------------|-------------|---------------------------------------------------------|
-| --threadfqdump  |             | Runs multi-threading for fastq-dump for sra processing. |
+4. Start running your own analysis!
 
-### QC Options
+   ```bash
+   nextflow run nf-core/nascent --input samplesheet.csv --outdir <OUTDIR> --genome GRCh37 -profile <docker/singularity/podman/shifter/charliecloud/conda/institute>
+   ```
 
-| Arguments       | Usage       | Description                                             |
-|-----------------|-------------|---------------------------------------------------------|
-| --skipMultiQC   |             | Skip running MultiQC.                                   |
-| --skipRSeQC     |             | Skip running RSeQC.                                     |
+## Documentation
+
+The nf-core/nascent pipeline comes with documentation about the pipeline [usage](https://nf-co.re/nascent/usage), [parameters](https://nf-co.re/nascent/parameters) and [output](https://nf-co.re/nascent/output).
 
 ## Credits
+
 nf-core/nascent was originally written by Ignacio Tripodi ([@ignaciot](https://github.com/ignaciot)) and Margaret Gruca ([@magruca](https://github.com/magruca)).
 
-Many thanks to the nf-core team and all who provided invaluable feedback and assistance along the way, particularly to [@apeltzer](https://github.com/apeltzer), [@ewels](https://github.com/ewels), [@drpatelh](https://github.com/drpatelh), and [@pditommaso](https://github.com/pditommaso).
+The pipeline was re-written in Nextflow DSL2 by Edmund Miller ([@Emiller88](https://github.com/emiller88)) and Sruthi Suresh ([@sruthipsuresh](https://github.com/sruthipsuresh)) from [The Functional Genomics Laboratory](https://taehoonkim.org/) at [The Univeristy of Texas at Dallas](https://www.utdallas.edu/)
+
+We thank the following people for their extensive assistance in the development of this pipeline:
+
+[@apeltzer](https://github.com/apeltzer)
+[@ewels](https://github.com/ewels)
+[@drpatelh](https://github.com/drpatelh)
+[@pditommaso](https://github.com/pditommaso)
+[Tae Hoon Kim](https://github.com/taehoonkim-phd)
+[@easterwoods](https://github.com/easterwoods)
+
+## Contributions and Support
+
+If you would like to contribute to this pipeline, please see the [contributing guidelines](.github/CONTRIBUTING.md).
+
+For further information or help, don't hesitate to get in touch on the [Slack `#nascent` channel](https://nfcore.slack.com/channels/nascent) (you can join with [this invite](https://nf-co.re/join/slack)).
+
+## Citations
+
+<!-- TODO nf-core: Add citation for pipeline after first release. Uncomment lines below and update Zenodo doi and badge at the top of this file. -->
```

Comment:
Since there is a previous release, there should be a zenodo link around that should be added here

Reply:
[![DOI](https://zenodo.org/badge/157735234.svg)](https://zenodo.org/badge/latestdoi/157735234)